.idea/gradle.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove the idea project files - probably we should add .idea to gitignore :-)